### ISSUE TITLE
Research - idme and logingov SiS errors

### ISSUE THREAD

**Issue by joeniquette** [department-of-veterans-affairs/va.gov-team#42480]
Research what errors each CSP may send to us that we need to handle differently than we already do. 

Acceptance Criteria:

- [x] Document all the possible errors from each CSP as it relates to calls we will make for SiS
- [x] Publish documentation to private github folder
- [x] Create new issue with to-do list of code changes required

**Comment by joeniquette** [comment]
Document published: https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/Identity/Sign%20In%20Service/Errors/IDme/external.md

**Comment by nsoutouras** [comment]
@nsoutouras bug Jeff Holden about this

**Comment by joeniquette** [comment]
OIDC Error source code from Logingov: https://github.com/18F/identity-idp/blob/main/config/locales/openid_connect/en.yml

Still need remaining redirect errors from them.

**Comment by nsoutouras** [comment]
Still need confirmation from ID.me before we can close
